cl/corpus_importer/management/commands/import_tn.py

```diff
@@ -0,0 +1,188 @@
+import itertools
+import json
+import logging
+from glob import glob
+
+from dateutil import parser
+from django.utils.encoding import force_bytes
+
+from cl.lib.command_utils import VerboseCommand
+from cl.lib.crypto import sha1
+from cl.people_db.models import Person
+from cl.scrapers.management.commands.cl_scrape_opinions import (
+    make_objects,
+    save_everything,
+)
+from cl.scrapers.tasks import extract_doc_content
+from cl.search.models import Court, Docket, Opinion
+
+
+def make_item(case):
+    """Make an import item for our save everything function
+
+    :param case: Case data received from the court.
+    :return: Case information.
+    :type: dict
+    """
+
+    judges = case["judge"]
+    first_name = judges.split(" ")[0].title()
```

Comment:
It's probably fine here, but I almost never use Python's `title` util. We have one in Juriscraper that handles things much better with things like `MacDonald`, `L'Oreal`, etc. I think it's called titlecase().

Reply:
Yeah, I like title case but knowing the data I knew there were no edge cases here.